Remove gendered language and americanize spellings (#571)
snowplow/documentation: docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracking-events/index.md

### Activity Tracking: page pings

> As well as tracking page views, we can monitor whether a user continues to engage with a page over time, and record how they digest content on the page over time.

Reviewer: ```suggestion
As well as tracking page views, we can monitor whether users continue to engage with pages over time, and record how they digest content on each page over time.
```
